packages/css/src/plugins/expand-shorthands/utils.tsx

```diff
@@ -0,0 +1,78 @@
+import { Node } from 'postcss-values-parser';
+
+/**
+ * Common global values
+ */
+export const globalValues = ['inherit', 'initial', 'unset'];
+
+/**
+ * Returns `true` if the node is a color,
+ * else `false`.
+ *
+ * @param node
+ */
+export const isColor = (node: Node) => {
+  if (node.type === 'word' && node.isColor) {
+    return true;
+  }
+
+  if (node.type === 'func' && node.isColor) {
+    return true;
+  }
+
+  return false;
+};
```

Comment:
nit: Can be simplified to 
```suggestion
const { isColor, type } = node;

return isColor && ['word', 'func'].includes(type);
```

Reply:
cool!

Reply:
update: i can't change it to this because of type restrictions (is color is only on two types of nodes)

i slightly made the code better though